src/client/Client.cc

```diff
@@ -1973,6 +1974,11 @@ int Client::make_request(MetaRequest *request,
       session = mds_sessions.at(mds);
     }
 
+    if (feature_needed != ULONG_MAX && !session->mds_features.test(feature_needed)) {
+      request->abort(-CEPHFS_EOPNOTSUPP);
```

Comment:
Circling back after discussion with @mchangir: I think we may want `-ENODATA` here. That is the correct error code for `getxattr(2)`. There is no ambiguity as the errno would be the same before and after the presence of `getvxattr`.

Reply:
Sure, will fix it.

Reply:
Done!